_forms/stance.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: form
+title: "Stance"
+title_mark: Stance
+published_date: 2025-11-06
+refactored_date: 2025-11-07
+authors:
+  - michael-basil
+  - kyle-ingersoll
+excerpt: "A Stance is the personal expression of one's FEBI™ energy archetypes through breath, tone, timing, and attention."
+---
+
+## Intention
+
+A Stance is the personal expression of one's FEBI™ energy archetypes through breath, tone, timing, and attention. In short, one's attitude during the moments that matter. Different people have different Stances, and Stances can be shifted. Apply both the Platnum and Golden Rules when shifting Stances.
```

Comment:
In relation to a context (others in the interaction and shared energy field)

(I like the call-out to platinum and golden rules